plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/impl/DataFrameAdapter.kt

```diff
@@ -35,19 +35,20 @@ fun SimpleCol.asDataColumn(): DataColumn<*> {
     return column
 }
 
-private fun List<AnyCol>.mapBack(): List<SimpleCol> =
-    map {
-        when (it) {
-            is ColumnGroup<*> -> {
-                SimpleColumnGroup(it.name(), it.columns().mapBack())
-            }
-
-            is FrameColumn<*> -> {
-                SimpleFrameColumn(it.name(), it[0].columns().mapBack())
-            }
-
-            else -> {
-                SimpleDataColumn(it.name(), it[0] as TypeApproximation)
-            }
+private fun List<AnyCol>.mapBack(): List<SimpleCol> = map { it.asSimpleColumn() }
+
+fun AnyCol.asSimpleColumn(): SimpleCol {
```

Comment:
could you add some comment, when to use this function

Reply:
sure